aspnetcore/signalr/hubs.md

```diff
@@ -90,6 +90,22 @@ To make calls to specific clients, use the properties of the `Clients` object. I
 
 [!code-csharp[Send messages](hubs/sample/hubs/chathub.cs?range=15-24)]
 
+## Strongly typed Hubs
+
+The drawback with using `SendAsync` is that it relies on a magic string to provide the method to be called on the client. This leaves code open to runtime errors if the method name is misspelled or missing from the client.
+
+An alternative to using `SendAsync` is to strongly type your `Hub` with `Hub<T>`. In the following example, the `ChatHub` client methods have been extracted out into an interface called `IChatClient`.  
+
+[!code-csharp[Interface for IChatClient](hubs/sample/hubs/ichatclient.cs?range=3-7)]
```

Comment:
Replace the `?range=3-7` portion of this with `?name=snippet_IChatClient`. This pulls in the named region defined in ichatclient.cs. This makes the include less brittle, as line numbers will change as the sample app evolves.